### DISCUSSION TITLE
Low-overhead version of BetterSEQTA+

### DISCUSSION THREAD

**Discussion by Crazypersonalph** [SethBurkart123/EvenBetterSEQTA#59] [Ideas]
Good afternoon everyone,
Here I am putting forward the suggestion/tracker of a low-overhead version of BetterSEQTA+

I noticed that upon loading BetterSEQTA+, it felt really quite bogged down compared to HyperSEQTA or the original BetterSEQTA, probably because of the new computing-intensive animations and themes.

I would like to propose a new setting which would allow the user to remove all non-essential compute-intensive tasks, such an animated backgrounds, themes, flashy animations, etc.

I believe we could also do a lot of optimisation in order to minimize overhead, which we can gradually work on as time goes on.

Kind Regards

**Comment by MEGA-Dawg68** [comment]
That would be nice. A light weight mode would definetly be useful for the people who have schools that give them expensive paperweights (like mine)

**Comment by SethBurkart123** [comment]
I definitely agree, that's a great idea just to give them that option, because the changes have definitely increased overhead a lot.

**Comment by SethBurkart123** [comment]
Just a bit of a step toward it. I made a change so that the background image iframe doesn't get created if there isn't any image selected (e8e8ec0) - and of course it automatically adds and removes when you remove the background. Hopefully it makes a little difference.

**Comment by SethBurkart123** [comment]
I think some other things we can do to improve performance is remove a lot of the repetitive code. If we made global variables for the content script that manage information like settings etc. you could cut down on requests to databases and extra code that introduces bugs 👍.
